cmd/snap-repair/runner.go

```diff
@@ -291,6 +420,19 @@ const (
 	DoneStatus
 )
 
+func (rs RepairStatus) String() string {
+	switch rs {
+	case RetryStatus:
+		return "retry"
+	case SkipStatus:
+		return "skip"
+	case DoneStatus:
+		return "done"
+	default:
+		panic(fmt.Sprintf("unknown repair status %d", rs))
```

Comment:
Should we return something like "unknown" instead? I worry about the potential of going forward into a core that understands more states, and reverting into one that doesn't.